assets/base_views.py

```diff
@@ -8,11 +8,11 @@ class ExampleForm(forms.Form):
 
     example_select = forms.ChoiceField(
         choices=[
-            ["", "Please select a language"],
-            ["english", "English"],
-            ["french", "French"],
-            ["german", "German"],
-            ["spanish", "Spanish"],
+            ("", "Please select a language"),
```

Comment:
ugh, we pull this in from job-server, and we don't need it

Can we ignore assets/*? we don't ahve any code we author in there?

We could perhaps remove this and some other python files